src/components/treeInfo/index.tsx

```diff
@@ -21,6 +22,18 @@ const ToggleTextButton = styled(Button)`
   padding: 0px;
 `;
 
+const UnadoptButton = styled(Button)`
+  && {
+    background-color: white;
+    border: 1px solid red;
+    color: red;
+  }
+
+  & :hover {
+    background-color: #fff1f1;
+  }
+`;
```

Comment:
To get rid of the `&&`, how about something like this?

```suggestion
const UnadoptButton = styled(Button)`
  & :hover {
    background-color: #fff1f1;
  }
`;
```

and then change `type="primary"` on line 127 to `danger`